docs/walkthroughs/setting-an-external-dmz-url.adoc

```diff
@@ -0,0 +1,36 @@
+[[setting-an-external-dmz-url]]
+= Setting an external DMZ URL
+
+== Overview
+
+Often in an enterprise setup, our infrastructure and services sit on internal networks that can not be accessed directly from the outside world. This means that the mobile client we create can only access services we provision by being on the internal network or by using some form of proxy. Most often these proxies live in something known as the DMZ or demilitarized zone. Genrally the DMZ is where all incoming traffic is handled and our services exposed. When talking to a proxy the URLs you use to configure your mobile client applicaiton, will be different than those of the services on the internal network. Setting the DMZ url on your mobile client within the OpenShift UI or via the mobile CLI will allow you to get an updated mobile client configuration that uses the external DMZ url instead of the internal service routes.
+
+
+=== Setting the DMZ URL
+
+You can set the DMZ URL on your mobile client definition in the OpenShift UI or directly via the CLI
+
+==== Using the mobile CLI 
+
+`mobile set client <clientID> --patch='{"spec": {"dmzUrl": "www.dmz.com"}}' --namespace=myproject`
+
+You can also use the mobile CLI as a kubernetes plugin
+
+`oc plugin mobile set client <clientID> -- --patch='{"spec": {"dmzUrl": "www.dmz.com"}}'`
+
+=== Using the UI to set the DMZ URL 
+
+You can set the DMZ URL on an already created mobile client via the OpenShift UI by locating the mobile client you want to set the DMZ URL for in the project overview screen. Then: 
```

Comment:
I think change "already created" to "existing" as it reads better IMO but just a nit so ignore if you want